source/puml/S-ENDA-metadata-service-context-diagram.puml

```diff
@@ -15,8 +15,8 @@ Boundary(providers, "Providers") {
     Person(serviceprovider, "Service Provider")
 }
 
-Boundary(systems, "S-ENDA") {
-    System(senda, 'S-ENDA Discovery Metadata Service')
+Boundary(systems, "Discovery Metadata Service") {
```

Comment:
it's more than a discovery metadata service, so I don't think this name works here. Also, I started a revision in https://github.com/metno/S-ENDA-documentation/pull/232 - could you look at that first?